### DISCUSSION TITLE
Should I use `useXXXContext`?

### DISCUSSION THREAD

**Discussion by m-shaka** [chakra-ui/ark#2365] [General]
Hi! I found `@ark-ui/react` exports hooks like `useComboboxContext` or `usePopoverContext`.
They are convenient, but there seems to be no documentation about them.
Are they stable APIs that I'm supposed to use?

**Comment by cschroeter** [comment]
@m-shaka 

Thanks for taking your time to create this discussion. The hooks are stabled and can be used. 
In the next version you can even use dedicated context components to work with the underlying  api.

See this [tweet](https://twitter.com/grizzly_codes/status/1775880898434441503)

**Reply by m-shaka** [reply]
Context components sound really nice! I appreciate your quick response
